### PR TITLE
Add add/rm-ing without name identifier

### PR DESCRIPTION
**Pull request by Cretezy**

### CODE REVIEW

**Comment by mfaerevaag** [comment]
Hi there! Is there a specific reason why you want this feature? As, at least I, seldom add new warp points it doesn't like too much of a convenience.

**Comment by Cretezy** [comment]
When adding many wrap points, simply `wd add` is much quicker than `wd add <spend a few seconds finding/typing a name>`. It's not the end of the world, but it's a little more convenient.

**Comment by mfaerevaag** [comment]
I think we will hold out one merging this for the moment, and see if there are more people wanting this feature. 

We appreciate you taking the time to submit this any how!

**Comment by raxod502** [comment]
I would love this feature. See also comments by @altschuler and @sindrenm on #44.

**Comment by Cretezy** [comment]
https://github.com/Cretezy/wd

**Comment by raxod502** [comment]
:+1: but it would be even cooler to get this merged into the official repository!

**Comment by mfaerevaag** [comment]
The people has spoken! Before we can merge this we need to fix the test suite, and maybe add an extra test for the new functionality. I hope to finish this during the weekend, and I'll come back to you 👍

File: wd.sh [hunk 1]
```diff
@@ -183,7 +188,12 @@ wd_add()
 
 wd_remove()
 {
-    local point=$1
+    local point=$2
```
Comment (mfaerevaag) [inline wd.sh]: How come you changed from `$1` to `$2`?

**Comment by mfaerevaag** [comment]
Merged 👍 Thanks again, @Cretezy

**Pull request closed without merging**